src/components/Text.jsx

```diff
@@ -63,6 +64,17 @@ const Text =
         ${text.base};
       `}
 
+  ${
+  /** @param {TextProps} props */
+  props =>
+      props.xsm &&
+      css`
+        ${text.xl};
+        @media (max-width: 960px) {
+          ${text.sm};
+        }
+      `}
```

Comment:
I understand that xsm is: `sm` on small screens and `xl` on big screens, but I agree with @Israel-Laguan, xsm sounds like "eXtra SMall"